docker-compose.yml

```diff
@@ -1,18 +1,93 @@
 version: "3"
 
 services:
-  app:
-    image: oncokb/oncokb
+  # oncokb-core requires that a MySQL server is started and has the the oncokb data imported into the database.
+  # Reach out to contact@oncokb.org to get access to the data dump
+  oncokb-core:
+    image: oncokb/oncokb:3.9.5
     ports:
       - "8080:8080"
     environment:
-      - "JAVA_OPTS=-Djdbc.driverClassName=com.mysql.jdbc.Driver -Djdbc.url=jdbc:mysql://db:3306/oncokb?useUnicode=yes&characterEncoding=UTF-8&useSSL=false -Djdbc.username=root -Djdbc.password=test"
+      JAVA_OPTS: >
+        -Djdbc.driverClassName=com.mysql.jdbc.Driver
+        -Djdbc.url=jdbc:mysql://host.docker.internal:3306/oncokb?useUnicode=yes&characterEncoding=UTF-8&useSSL=false
```

Comment:
Should we use a pseudo url for jdbc connection?

Reply:
Replaced with `-Djdbc.url=jdbc:mysql://[hostname]:3306/[database name]?useUnicode=yes&characterEncoding=UTF-8&useSSL=false`